src/components/Splash.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 
-import React from "react"
-import styled from "styled-components"
+import React, { useEffect } from "react"
+import styled, { createGlobalStyle } from "styled-components"
```

Comment:
Don't need this if it's a `styled-component`

```suggestion
import styled from "styled-components"
```